tests/models/cohere/test_modeling_cohere.py

```diff
@@ -0,0 +1,320 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the PyTorch Cohere model. """
+
+import unittest
+
+from parameterized import parameterized
+
+from transformers import CohereConfig, is_torch_available
+from transformers.testing_utils import (
+    require_torch,
+    slow,
+    torch_device,
+)
+
+from ...test_configuration_common import ConfigTester
+from ...test_modeling_common import ids_tensor
+
+
+if is_torch_available():
+    import torch
+
+    from transformers import CohereForCausalLM, CohereModel
+
+
+class CohereModelTester:
```

Comment:
```suggestion
# Copied from transformers.tests.models.llama.test_modeling_llama.LlamaModelTester with Llama->Cohere
class CohereModelTester:
```
Can we add also copied from on tests as well ?

Reply:
There are differences in the tests.